### DISCUSSION TITLE
add data to projected tile layer

### DISCUSSION THREAD

**Discussion by rh-afk** [opengeos/leafmap#502] [Q&A] [answered]
Hi every one to add data(dataframe or point) to custom projected tile layer. I can add custom tile from this example
[https://github.com/jupyter-widgets/ipyleaflet/blob/master/examples/CustomProjections.ipynb](url)
===========================================================================
my_projection = {
    "name": "EPSG:2163",
    "custom": True,  # This is important, it tells ipyleaflet that this projection is not on the predefined ones.
    "proj4def": "+proj=laea +lat_0=45 +lon_0=-100 +x_0=0 +y_0=0 +a=6370997 +b=6370997 +units=m +no_defs",
    "origin": [-2215235.82, 420644.41],
    "bounds": [[-8046094.81, 1819060.18], [988364.71, 3511186.72]],
    "resolutions": [8192.0, 4096.0, 2048.0, 1024.0, 512.0, 256.0],
}

wms = WMSLayer(
    url="https://ahocevar.com/geoserver/wms",
    layers="ne:NE1_HR_LC_SR_W_DR",
    format="image/png",
    transparent=True,
    min_zoom=0,
    attribution="ahocevar geospatial",
    crs=my_projection,  # I'm asking this WMS service to reproject the tile layer using EPSG:2163
)

m3 = Map(center=(40, -104), zoom=0, layers=(wms,), crs=my_projection)
================================================================================

using leafmap I just change :
m.add_tile_layer(
    url="https://ahocevar.com/geoserver/wms",
    layers="ne:NE1_HR_LC_SR_W_DR",
    format="image/png",
    transparent=True,
    min_zoom=0,
    attribution="ahocevar geospatial",
    crs=my_projection,  # I'm asking this WMS service to reproject the tile layer using EPSG:2163
)

Now the question is how i can plot data to that map because my data use different default base map projection 

thanks

**Comment by giswqs** [comment, marked as answer]
See https://github.com/opengeos/leafmap/discussions/215
